tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py

```diff
@@ -220,14 +251,21 @@ def get_brick_utilization(self):
                 {}
             ).iteritems():
                 for brick in sub_volume_bricks:
-                    brick_hostname = brick['hostname']
                     # Check if current brick is from localhost else utilization
                     # of brick from some other host can't be computed here..
+                    brick_hostname = tendrl_glusterfs_utils.find_brick_host(
+                        self.etcd_client, brick['hostname']
+                    )
+                    if brick_hostname:
```

Comment:
again, this piece of code seems common, please dont duplicate this everywhere use single function